packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -526,6 +526,11 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
     }
   }
 
+  // Whether or not the current positions should be reset
+  private _didPositionPropsChanges(newProps: ICalloutProps, oldProps: ICalloutProps): boolean {
+    return (!newProps.hidden && newProps.hidden !== this.props.hidden) || newProps.directionalHint !== this.props.directionalHint;
```

Comment:
why are you passing in oldProps if you're just going to use 'this'

Reply:
Whoops, good catch